exercises/01_intro/demo.R

```diff
@@ -0,0 +1,77 @@
+# Demo script for introduction session on 2021-09-16
+# We will transform Dotstat data to create figure 2.11 in OECD RCaG 2020
+# Disclaimer: This is an example for learning purposes, 
+# there will be some differences with the original figure
+
+## 0 Packages ----------------------------------------------------------------------------------------------------------
+
+# install.packages('tidyverse')
+library(tidyverse) # includes dplyr, tidyr, readr, ...
+# install.packages("writexl")
+library(writexl) # writing Excel files
+
+
+## 1 Data ----------------------------------------------------------------------------------------------------------
+## 2.1 Import ----------------------------------------------------------------------------------------------------------
+
+data <- read_csv("gva_per_worker.csv")
```

Comment:
This data raises a lot of parsing failures, and it's the type of warning that it's better to not get too used to ignoring. I am going to add a bit defining the column specification via `col_types` so the warning doesn't show up f9998720388ab78b2dd465e58d08187084e76af9